### DISCUSSION TITLE
v0.0.8

### DISCUSSION THREAD

**Discussion by SWHL** [SWHL/RapidVideOCRDesktop#2] [Announcements]
Add to support English version.

**Full Changelog**: https://github.com/SWHL/RapidVideOCRDesktop/compare/v0.0.7...v0.0.8

<hr /><em>This discussion was created from the release <a href='https://github.com/SWHL/RapidVideOCRDesktop/releases/tag/v0.0.8'>v0.0.8</a>.</em>

**Comment by freddiehdxd** [comment]
doesn't open

**Reply by SWHL** [reply]
Please provide more information.

**Reply by freddiehdxd** [reply]
when l try to run it, it just disappear l had the previews version and work well

**Reply by SWHL** [reply]
Does that means your problem has been solved?

**Comment by freddiehdxd** [comment]
The software for pc is not working when we try to run it nothing happens

**Reply by SWHL** [reply]
If you know how to run python script, you could directly run the [RapidVideOCR](https://github.com/SWHL/RapidVideOCR) project.

The Desktop version of RapidVideOCR has poor compatibility.
